handel/src/services/ecs-fargate/params-schema.json

```diff
@@ -0,0 +1,300 @@
+                        }
+                    }
+                }
+            },
+            "required": [
+                "min_tasks",
+                "max_tasks"
+            ],
+            "additionalProperties": false,
+            "errorMessage": {
+                "required": {
+                    "min_tasks": "The 'min_tasks' field is required in the 'auto_scaling' section",
+                    "max_tasks": "The 'max_tasks' field is required in the 'auto_scaling' section"
+                },
+                "additionalProperties": "Invalid/Unknown property specified in the 'auto_scaling' section"
+            }
+        },
+        "load_balancer": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "description": "The accepted values are 'http' and 'https'",
+                    "pattern": "^http|https$",
+                    "errorMessage": "The 'type' field must be 'http' or 'https' in the 'load_balancer' section"
+                },
+                "timeout": {
+                    "type": "integer",
+                    "errorMessage": "The 'timeout' field must be an integer in the 'load_balancer' section"
+                },
+                "https_certificate": {
+                    "type": "string",
+                    "errorMessage": "The 'https-certificate' field must be a string in the 'load_balancer' section"
+                },
+                "dns_names": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "errorMessage": "The 'dns_names' field must be an array of strings in the 'load_balancer' section"
+                }
+            },
+            "required": [
+                "type"
+            ],
+            "additionalProperties": false,
+            "errorMessage": {
+                "required": {
+                    "type": "The 'type' field is required in the 'load_balancer' section"
+                },
+                "additionalProperties": "Invalid/Unknown property specified in the 'load_balancer' section"
+            }
+        },
+        "log_retention_in_days": {
+            "type": "number",
+            "errorMessage": "The 'log_retention_in_days' parameter must be a number"
+        },
+        "tags": {
+            "type": "object",
+            "description": "An arbitrary list of key/value pairs to be added as tags to the service",
+            "patternProperties": {
+                "^.*$": {
+                    "anyOf": [
+                        {"type": "string"},
+                        {"type": "number"}
+                    ]
+                }
+            },
+            "errorMessage": "The 'tags' element must contain 1 or more simple key/value pairs where the values are strings or numbers",
+            "additionalProperties": false
+        }
```

Comment:
You'll want to add the 'dependencies' section in the schema file too, since Fargate can have dependencies.